### PR TITLE
Add ResourceLoader: only start class after loading resources.

### LINKED ISSUE #19: Sometimes lesson 1 opens without the robot
This happens because the image may load after the canvas has been drawn. A fix would be to only start the lesson after all resources load.

This sometimes also used to happen to the "Play" button.

### PR DESCRIPTION
**Pull request by gpoesia**
Fixes #19 by loading images before starting the class.

By creating a virtual element before, we force the browser to load the image to its local cache, and it will be stored locally by the second time it's loaded from within the class.

### CODE REVIEW

File: src/app.js [hunk 1]
```diff
@@ -8,8 +8,17 @@ var DummyLesson = require("./lesson/dummy_lesson");
 var Lesson01 = require("./lesson/lesson01");
 var LessonEnvironment = require("./view/lesson_environment");
 
-var startLesson = function(lessonClass, domElement) {
-  ReactDOM.render(<LessonEnvironment lesson={new lessonClass()} />,
+var startLesson = function(lesson, domElement) {
+  var lessonInstance = null;
+  if (typeof(lessonClass) === 'function') {
```
Comment (vrod) [inline src/app.js]: Is this right? I don't see `lessonClass` defined anywhere. I am assuming it should be passed as a parameter.

**Comment by vrod** [comment]
There is one strange thing that I noticed, and commented on the respective line. The rest looks good. After taking a look into that, feel free to merge.

**Comment by gpoesia** [comment]
Right, that's odd. I noticed it when merging all pull requests for yesterday's nightly release. Thanks, I'll fix it and merge it.

**Comment by gpoesia** [comment]
Merged (333a4e2381edc7d92524c8cdb152028b20c390c4).

**Pull request closed without merging**